libmscore/measure.cpp

```diff
@@ -1734,7 +1734,7 @@ void Measure::read(XmlReader& e, int staffIdx)
                   breath->setTrack(e.track());
                   int tick = e.tick();
                   breath->read(e);
-                  if (score()->mscVersion() < 205) {
+                  if (score()->mscVersion() <= 206) {
                         // older scores placed the breath segment right after the chord to which it applies
```

Comment:
I think the problem is what is described in the comment here. The current selectionfilter13.mscx file was version 200, so it was less 205, it maybe should be renewed.